generators/cleanup.js

```diff
@@ -298,7 +298,7 @@ function cleanupOldFiles(generator) {
       generator.removeFile(`${CLIENT_TEST_SRC_DIR}spec/app/shared/util/entity-utils.spec.ts`);
       generator.removeFile(`${CLIENT_TEST_SRC_DIR}spec/app/shared/auth/private-route.spec.tsx`);
       generator.removeFile(`${CLIENT_TEST_SRC_DIR}spec/app/shared/error/error-boundary.spec.tsx`);
-      generator.removeFile(`${CLIENT_TEST_SRC_DIR}spec/app/shared/error/error-boundary-route.spec.tsx`);
+      generator.removeFile(`${CLIENT_TEST_SRC_DIR}spec/app/shared/error/error-boundary-routes.spec.tsx`);
```

Comment:
I made the requested changes.

I don't fully get what the code is doing. Why do we check if the JHipster version is **below** 7.8.2? In my mind, it seems to make more sense to check if we are after 7.8.1 and only then delete these files since they are only needed up to version 7.8.1.
And why was only the spec file removed before, while you suggest also removing the component?

Reply:
I figured out the answer to the second question: The test files were relocated next to the components they are testing and therefore the test files at the old location needed to be deleted.

Still haven't figured out the answer to the first question. @mshima or @DanielFran Could you help me understand?